test/dlc_tests/container_tests/bin/benchmark/get_mxnet_avg_speed.py

```diff
@@ -0,0 +1,27 @@
+# usage: python <script_name>.py <log_file> 4 Training & Inference GPU
+# usage: python <script_name>.py <log_file> 1 Inference CPU
+import sys
```

Comment:
Rename to: _throughput. `throughput` is a better term to major performance in samples/sec.

Reply:
Will do it. Thanks